en/docs/develop/authentication-data-api.md

```diff
@@ -41,15 +41,15 @@ need to configure the Identity Server as follows.
         <tbody>
             <tr>
                 <td>filter_policy</td>
-                <td>Value is either <b>include</b> or <b>exclude</b>. An include indicates a whitelist, whereas an <b>exclude</b> indicates a blacklist.</td>
+                <td>Value is either <b>include</b> or <b>exclude</b>. An include indicates an allowlist, whereas an <b>exclude</b> indicates a denylist.</td>
```

Comment:
WDYT about:

"An include indicates an addition to the allowlist, whereas an exclude indicates an addition to the denylist"

Reply:
I guess this is confusing since this config's function is not to add to the allowlist. It is merely to filter the components in the allowlist.